python/tvm/relay/frontend/pytorch.py

```diff
@@ -4291,7 +4291,11 @@ def _handel_nested_input(inputs):
 
             self.current_op.pop()
 
-        return [_wrap_const(outputs[ret_name]) for ret_name in ret_names]
+        return [
+            _wrap_const(outputs[ret_name])
+            for ret_name in ret_names
+            if ret_name != "aten::adaptive_max_pool1d_0_1"
```

Comment:
LGTM, but please set TODO somewhere near. Due to it is still workaround for specific op on high-level API and we do not miss it for more correct fix in the future.

Reply:
I found that all TODO in python requires @someone. In this issue, who should I @?

Reply:
Of course your-self) In general It does not matter due to it does not mean that the man who wrote TODO should fix it, it is a tag for anybody to understand that there is some issue